---
 .../HostWorkspace/AutoLoadProjectsInitializer.cs     | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/AutoLoadProjectsInitializer.cs b/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/AutoLoadProjectsInitializer.cs
index db534b04ebf..096aa8726d4 100644
--- a/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/AutoLoadProjectsInitializer.cs
+++ b/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/AutoLoadProjectsInitializer.cs
@@ -86,6 +86,7 @@ public async Task OnInitializedAsync(ClientCapabilities clientCapabilities, Reque
             if (TryGetFolderPath(folder, _logger, out var folderPath))
             {
                 projectFiles.AddRange(Directory.EnumerateFiles(folderPath, "*.csproj", SearchOption.AllDirectories));
+                projectFiles.AddRange(Directory.EnumerateFiles(folderPath, "*.vbproj", SearchOption.AllDirectories));
             }
         }
 
-- 
2.53.0.windows.3
